catext/ssp-coop.md

```diff
@@ -5,5 +5,5 @@
 
 [Back](../README.md)
 
-* All [SPP Solo rules](ssp.md) apply to this category.
+* All [SSP Solo rules](ssp.md) apply to this category.
```

Comment:
It's pretty odd that we don't seem to define "SSP" or what it is (Set Seed Peaceful) here. I'm not sure if this should be addressed in this change or not, but in any case we should follow it up, as I imagine the heavy use of abbreviations can be confusing